packages/server/graphql/public/mutations/updateSAML.ts

```diff
@@ -0,0 +1,36 @@
+import getRethink from '../../../database/rethinkDriver'
+import {MutationResolvers} from '../resolverTypes'
+import getSignOnURL from './helpers/SAMLHelpers/getSignOnURL'
+
+const updateSAML: MutationResolvers['updateSAML'] = async (_source, {orgId, metadata}, {}) => {
+  const r = await getRethink()
+
+  // RESOLUTION
+  const samlRecord = await r
+    .table('SAML')
+    .getAll(orgId, {index: 'orgId'})
+    .limit(1)
+    .nth(0)
+    .default(null)
+    .run()
+
+  if (!samlRecord) {
+    return {error: {message: 'SAML has not been created in Parabol yet.'}}
```

Comment:
+1 as this is something called by the user, I'd say we should log it to sentry, check the usage of `standardError` https://github.com/ParabolInc/parabol/blob/a1ca44dda2254fe6747e64e2c358c9dd53f7237b/packages/server/utils/standardError.ts#L3